.coafile

```diff
@@ -26,6 +26,7 @@ bears = LineLengthBear
 
 [all.links]
 bears = InvalidLinkBear
+link_ignore_regex = ((github|gitlab)\.com|\{|\$)
```

Comment:
This is not what I said to do in https://github.com/coala/corobo/pull/573#discussion_r199987571

Here you are ignoring all links to those websites in any file.  Obviously not approriate I would think.